cpp/src/io/json/json_column.cu

```diff
@@ -393,6 +393,41 @@ std::vector<std::string> copy_strings_to_host(device_span<SymbolT const> input,
   return to_host(d_column_names->view());
 }
 
+rmm::device_uvector<uint8_t> is_all_nulls_str_column(device_span<SymbolT const> input,
```

Comment:
I think a doc string would be useful here. The name is a little unintuitive as well. It reads like "is this a string column that contains all nulls" but it's actually checking a series of columns.

Reply:
added doc string.